src/app/home/home.component.html

```diff
@@ -1,14 +1,28 @@
-<div id="container">
-
-    <label> User Name</label>
-    <input type="text"/>
-    <label> Password</label>
-    <input type="password"/>
-
-    <h3><a routerLink="/dashboard">Login</a></h3>
-    <h3><a routerLink="">Register</a></h3>
-
-</div>
-
-
+<body>
```

Comment:
remove body tag and corresponding css

Reply:
I removed body tag replaced with div tag and i can changed css also
![Screenshot (145)](https://github.com/user-attachments/assets/93584733-9a0e-4bd7-be06-fa28800fd529)